src/modules/mc_pos_control/MulticopterPositionControl.cpp

```diff
@@ -557,6 +561,74 @@ void MulticopterPositionControl::Run()
 			// Publish attitude setpoint output
 			vehicle_attitude_setpoint_s attitude_setpoint{};
 			_control.getAttitudeSetpoint(attitude_setpoint);
+
+			/*** CUSTOM ***/
+			if(_param_airframe.get() == 12 ){ //If tilting_multirotors
```

Comment:
I'd say the best way would be to have a different strategy to map acceleration to attitude altogether. At least thrust to attitude. Otherwise, there are assumptions involved that don't apply if you can tilt the thrust without the vehicle. I wanted to make the acceleration to attitude part more modular such that it's easy to plug in a different allocation but the limit feedback is what makes it not trivial. At least thrust to attitude is easy to replace in terms of interface.
https://github.com/PX4/PX4-Autopilot/blob/99197919d75b24519e4435e28b98dd000c9312aa/src/modules/mc_pos_control/PositionControl/PositionControl.cpp#L261

Reply:
I am not sure I completely understand what you mean.

> Otherwise, there are assumptions involved that don't apply if you can tilt the thrust without the vehicle

With omnidirectional tilting drones you can tilt the thrust by tilting the rotors, without tilting the vehicle